src/main/scala/com/microsoft/hyperspace/index/IndexLogEntry.scala

```diff
@@ -608,7 +609,30 @@ case class IndexLogEntryTag[T](name: String)
 object IndexLogEntry {
   val VERSION: String = "0.1"
 
+  val HYPERSPACE_PROJECT_VERSION: String = "hyperspaceVersion"
```

Comment:
Could you move this to IndexConstants? since other key strings are also in there.

Reply:
And it should be `HYPERSPACE_VERSION_PROPERTY` if we move this to `IndexConstants`.